### DISCUSSION TITLE
Export Intunewin file supported ?

### DISCUSSION THREAD

**Discussion by steve2vince** [Micke-K/IntuneManagement#185] [General]
Hi,
First of all, congratulation for this great job ! Very good tool.
My question is simple : is it possible to export the intunewin file when exporting an application ?
If yes, what is the procedure please ?
Thanks !

**Comment by Micke-K** [comment]
Hello!

Easy answer No. Complex answer Yes. 

You can download the file (with the tool) but it's encrypted and there's no api to get the encryption keys. 

There is a script in the latest release that can export the encryption keys from package intunewin files. 

There are blogs on how to get the encryption keys the hard way. You could use that in the combination of using the tool to downloading and automatically decrypt the intunewin file. 

You have some links a d information in Discussion 55. 

I hope that gives you something to go on. 

Cheers!
